stl/inc/algorithm

```diff
@@ -3665,9 +3669,8 @@ template <class _BidIt, class _Pr>
 void _Insertion_sort_isort_max_chunks(_BidIt _First, const _BidIt _Last, _Iter_diff_t<_BidIt> _Count, _Pr _Pred) {
     // insertion sort every chunk of distance _ISORT_MAX in [_First, _Last)
     // pre: _Count == distance(_First, _Last)
-    constexpr auto _Diffsort_max = static_cast<_Iter_diff_t<_BidIt>>(_ISORT_MAX);
-    for (; _Diffsort_max < _Count; _Count -= _Diffsort_max) { // sort chunks
-        _First = _Insertion_sort_unchecked(_First, _STD next(_First, _Diffsort_max), _Pred);
+    for (; _Count <= _ISORT_MAX; _Count -= _ISORT_MAX) { // sort chunks

```

Comment:
Did you intend to invert the condition? This is the complement of the prior condition `_Count > _ISORT_MAX`.

Reply:
Dear lord :sigh: